### ISSUE TITLE
Confirmation alert when the tab is closed while training

### ISSUE THREAD

**Issue by GuilhermeCaeiro** [m4nuC/aifiddle-issues#4]
Right now, if the tab is closed while a training procedure is going on, the application doesn't provide any alert. I think that the best behavior would be to alert the user about the ongoing training and ask for a confirmation (like Facebook does when you have an unsent message/comment). The objective of that change is to avoid an accidental closing.

**Comment by m4nuC** [comment]
Good point. I will take care of this during next update. (early next week)
